src/main/groovy/com/bmuschko/gradle/docker/tasks/network/DockerRemoveNetwork.groovy

```diff
@@ -0,0 +1,10 @@
+package com.bmuschko.gradle.docker.tasks.network
+
+class DockerRemoveNetwork extends DockerExistingNetwork {
+    @Override
+    void runRemoteCommand(final dockerClient) {
+        final networkId = getNetworkId()
+        logger.quiet "Removing network '$networkId'."
+        dockerClient.removeNetworkCmd(networkId).exec()
```

Comment:
imo, that wouldn't be clean because `onNext` on Observables etc should usually carry a meaningful value.
You can already get success feeback from `onComplete`, right?